pkg/investigations/clusterhealthcheck/clusterhealthcheck.go

```
@@ -0,0 +1,1155 @@
+
+	return latestMinor, nil
+}
+
+// channelHasVersions queries the OpenShift upgrade graph API and returns whether
+// the given channel has any versions available.
+func (d *defaultEOLChecker) channelHasVersions(apiURL string) (bool, error) {
+	resp, err := http.Get(apiURL) //nolint:gosec
+	if err != nil {
+		return false, err
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)